app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return if user.nil?
+
+    @user = user
+    registered_user
+  end
+
+  def registered_user
+    can :manage, Expense
```

Comment:
I don't know how this got here... It's PR https://github.com/groupbuddies/botler/pull/2, I must have accidentally added the file to this branch. :(